modules/custom/az_flexible_block/src/Plugin/Menu/LocalAction/AzFlexibleBlockAddLocalAction.php

```diff
@@ -0,0 +1,26 @@
+<?php
+
+namespace Drupal\az_flexible_block\Plugin\Menu\LocalAction;
+
+use Drupal\Core\Menu\LocalActionDefault;
+use Drupal\Core\Routing\RouteMatchInterface;
+use Drupal\Core\Url;
+
+/**
+ * Modifies the 'Add custom block' local action.
+ */
+class AzFlexibleBlockAddLocalAction extends LocalActionDefault {
+
+  /**
+   * {@inheritdoc}
+   */
+  public function getOptions(RouteMatchInterface $route_match) {
+    $options = parent::getOptions($route_match);
+    // Adds a destination on custom block listing.
+    if ($route_match->getRouteName() == 'view.block_content.page_1') {
```

Comment:
```suggestion
    if ($route_match->getRouteName() == 'view.block_content.admin') {
```
@trackleft do I need to make this change too?